src/microsoft-trydotnet-editor/src/ProjectKernelWithWASMRunner.ts

```diff
@@ -219,7 +204,7 @@ export class ProjectKernelWithWASMRunner extends projectKernel.ProjectKernel {
   }
 
   private deriveToken(originalCommand: polyglotNotebooks.KernelCommandEnvelope) {
```

Comment:
Is this still used at all? This method should be deleted as we need not to manipulate command tokens.

Reply:
sure. I'll do that